changelog/1.19.0.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "1.19.0"
+description: "Released on 05/19/2021"
+---
+
+### Breaking changes ❗
+
+- infra: Workspace assets created by Coder that were previously located at
+  `/opt/coder`, such as code-server and coder-cli, have been moved to
+  `/var/tmp/coder`
+- infra: Workspaces as code templates have been updated from version _0.1_ to
+  _0.2_. Version _0.1_ templates will no longer work.
+- web: The workspace as code _Open in Coder_ embeddable button flow no longer
+  includes a _clone_ step; buttons created from prior versions will still work,
+  but Coder will not clone the project. To automate project cloning, use
+  [`workspace.configure.start`](../workspaces/workspaces-as-code/templates.md#workspaceconfigurestart)
+- web: The embedded form for workspaces as code located at **Manage** >
+  **Admin** > **Templates** no longer uses the following fields: **Project
+  Repository URL** and **Project Git Service**
+
+### Features ✨
+
+- web: Introduced resource quotas to organizations. Resource quotas define the
+  maximum resource utilization allowable for each member within an organization
+  for which such quotas are enabled. Modify them at **Manage** >
+  **Organizations** > **Edit Organization**
+- web: Added a reactivation step for returning users that had been marked as
+  dormant. The reactivation step requires the user to agree to use a license
+  seat
+- web: Added an admin setting controlling the maximum dev URL access level
+  across all workspaces within the deployment. Modify the maximum dev URL access
+  level at **Manage** > **Admin** > **Infrastructure** in the card titled **Dev
+  URL Access Permissions**
+- web: The CVM option will be selected by default when creating a custom
+  environment (if CVMs are enabled at the site level)
+- web: Improved display and support for errors returned by an Identity Provider
+  (IdP) during sign-in
+- infra: Changed permissions of `/home/coder/` from _755_ to _744_
+- infra: Workspace builds are now properly validated when setting policy
+  templates
+- code-server: Upgraded code-server to 3.10.0 (latest version)
+
+### Bug fixes 🐛
+
+- web: Fixed issue where the **Edit Workspace** dialog showed the resource
+  selectors as disabled while still allowing modifications
+- web: Fixed issue where the **Edit Workspace** dialog displayed resource
+  allocation incorrectly if the workspace was initially built using a template
+- web: Updated validation logic so that workspaces with invalid names can no
+  longer be created
+- web: Fixed issue causing web-based terminals to show two scrollbars and
+  overflowing content when system banners are present
+- infra: Fixed issue causing workspace autostart to stop or fail intermittently
+- web: Fixed issue in the **Audit Logs** preventing users from clearing the
+  filters
+- web: Added a _.csv_ extension to exported audit logs
```

Comment:
@khorne3 wondering if these should all be `Fixed an issue` instead of `Fixed issue`.

Reply:
I think I prefer the terseness of "Fixed issue"?

Reply:
Sounds good, I defer to you! Was a gut check, wasn't sure.